NDB.Covid19/NDB.Covid19.iOS/Views/AuthenticationFlow/QuestionnaireViewController.cs

```diff
@@ -170,6 +170,9 @@ void OnValidationFail()
         void OnFail()
         {
             NextBtn.HideSpinner();
+            AuthErrorUtils.GoToTechnicalError(this, LogSeverity.ERROR, null,
```

Comment:
and what with validation fail?

Reply:
It's only invoked if the sympton onset date is not selected, and in that case it displays a dialog.